lib/puppet/network/http/api/v3/indirection_type.rb

```diff
@@ -0,0 +1,13 @@
+class Puppet::Network::HTTP::API::V3::IndirectionType
+
+  INDIRECTION_TYPE_MAP = {
+    "certificate" => :ca,
+    "certificate_request" => :ca,
```

Comment:
I don't think so, because when this is used (https://github.com/puppetlabs/puppet/pull/3464/files#diff-20dc5688c93b62d47f42bf0a835001ecR83 and https://github.com/puppetlabs/puppet/pull/3464/files#diff-20dc5688c93b62d47f42bf0a835001ecR237), the indirection name passed in is singular.